imports/ui/report-explorer/report-explorer.jsx

```diff
@@ -51,11 +60,11 @@ class ReportExplorer extends Component {
   }
 
   makeReportGrouping = memoizeOne(
-    (reportList, filterStatus, myInvolvement) => {
-      const statusFilter = filterStatus
-        ? report => report.status !== REPORT_DRAFT_STATUS
-        : report => report.status === REPORT_DRAFT_STATUS
-      const creatorFilter = myInvolvement ? x => x.assignee === this.props.currentUser.bugzillaCreds.login : x => true
+    (reportList, statusFilterValues, roleFilterValues, sortBy) => {
+      const statusFilter = statusFilterValues === 'All' ? report => true
```

Comment:
Change this to a "switch" clause or a "if..else" chain. Nested ternary logic is too convoluted